shared/common-adapters/text.meta.desktop.js

```diff
@@ -28,7 +28,7 @@ function fontSizeToSizeStyle(fontSize: number): ?Object {
     '16': 20,
     '14': 18,
     '13': 17,
-    '11': 15,
+    '12': 16,
```

Comment:
Might be good to leave `'11': 15` in place (in addition to the new `'12': 16`), in case you missed any of the conversions.

Reply:
We likely prefer it crash